lib/utils/scrollTo.js

```diff
@@ -1,14 +1,26 @@
+
+import { detectBrowser } from './index.js';
+
 function scrollTo(element, elementBoundary, to, duration, callback) {
-  const start = element[elementBoundary],
-    change = to - start,
-    increment = 10;
+  const { name, version } = detectBrowser(navigator && navigator.userAgent);
+  const [ major ] = version.split('.');
+  const newChrome = (name === 'chrome' && Number(major) > 60);
+  const start = newChrome ? window.scrollY : element[elementBoundary];
+  const change = to - start;
+  const increment = 10;
 
   let currentTime = 0;
   let globalId = requestAnimationFrame(repeatOften);
   function repeatOften() {
     currentTime += increment;
     let val = easeInOutQuad(currentTime, start, change, duration);
-    element[elementBoundary] = val;
+
+    // canary + newer chrome
+    if (newChrome) {
+      window.scrollTo(0, val);
```

Comment:
This breaks horizontal scrolling functionality. The element + method should get passed into this function via the components that use it (body, scrollTop, scrollLeft, etc)